anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+from ..utilities import init_flask
+from flask_sqlalchemy import SQLAlchemy
+from ..models import InfographicsDataCache, InfographicsDataCacheTemp, NewsFlash
+from ..constants import CONST
+import anyway.infographics_utils
+import logging
+
+
+app = init_flask()
+db = SQLAlchemy(app)
+
+
+def get_infographics_data_from_cache(news_flash_id, years_ago):
+    db_item = (
+        db.session.query(InfographicsDataCache)
+        .filter(InfographicsDataCache.news_flash_id == news_flash_id)
+        .filter(InfographicsDataCache.years_ago == years_ago)
+        .first()
+    )
+    logging.debug(f"retrieved from cache {type(db_item)}:{db_item}"[:70])
+    db.session.commit()
+    try:
+        return db_item.get_data() if db_item else {}
+    except Exception as e:
+        logging.error(
+            f"Exception while extracting data from returned cache item flash_id:{news_flash_id},years:{years_ago})"
+            f"returned value {type(db_item)}"
+            f":cause:{e.__cause__}, class:{e.__class__}"
+        )
+        return {}
+
+
+def copy_temp_into_cache():
+    num_items_cache = db.session.execute("SELECT * from infographics_data_cache").rowcount
```

Comment:
why not select count(*) instead of select * ? (Relevant in additional locations as well)